packages/maskbook/src/plugins/PoolTogether/constants.ts

```diff
@@ -14,3 +15,5 @@ export const ONE_DAY_SECONDS = 86400
 export const ONE_WEEK_SECONDS = ONE_DAY_SECONDS * 7
 
 export const ADDRESS_ZERO = '0x0000000000000000000000000000000000000000'
+export const SECONDS_PER_BLOCK = 14
+export const WEI_PER_ETHER = new BigNumber(10).pow(18)
```

Comment:
I don't see these two constant being used. Do you forget to remove them?

Reply:
Yes, I forgot, removed them.